tests/helpers/is_cron_error.ts

```diff
@@ -0,0 +1,27 @@
+import { ExclusiveParametersError } from '../../src/errors';
+
+export function isCronError(error: unknown): boolean {
+	return (
+		error instanceof Error &&
+		(error instanceof ExclusiveParametersError ||
+			error.message === 'time must be an instance of CronTime.' ||
```

Comment:
is this a list of all errors across the codebase? is this something we'll have to maintain and update every time we change the text of one of the errors?

Reply:
yes, indeed we would need to update that list when adding/updating errors. a workaround could be to create a `CronError` class we would use everywhere and simply check if the error is an instance of it.

Reply:
that would be preferred. that way people can make text changes to errors without breaking the build. also would be good for the error on this test to say "make sure the relevant code is using an instance of `CronError`" so new contributors know how to fix it